### ISSUE TITLE
Move package files to subdirectory?

### ISSUE THREAD

**Issue by andreasnoack** [JuliaRegistries/General#17]
Things are not as bad here as in METADATA where the README is not visible, but if we add a README to this repo, it would still show up fairly low on the site. Might be better to put all the package file in a single directory such that the root directory has relatively few files.

**Comment by StefanKarpinski** [comment]
Sure, could be `packages/{A,B,...}/`. Can be done whenever since the location is included in the `Registry.toml` file. Kind of annoying that it makes every line in that file 9 characters longer though.

**Comment by andreasnoack** [comment]
This might not be as important as I initially thought it would be. When trying to look into this for METADATA I realized that GitHub had actually made a change such that the list was contracted and had a scrollbar. However, this was only the case for repos for which you didn't have write access so, initially, I didn't notice this as I was mainly looking at repos for which I had write access. This contraction also only happened if a repo had a README but the detection of the README only happened after the list had been limited to 1000 so it didn't work for METADATA. I reported this to GitHub Support and, as a consequence, they have temporarily disabled the contraction feature for the time being. Hopefully it will return.
